cgOne/blueprint.py

```diff
@@ -13,5 +13,60 @@ class Blueprint:
     """
 
     def __init__(self):
-        self.max_speed = 0
-        self.exhaust = 0
+    	self.max_speed = 0
+    	self.exhaust = 0
+    	self.type = 'Hatchback'
+    	self.exterior = 'Aluminum'
+    	self.interior = 'Fabric'
+    	self.make = 'Make'
+    	self.model = 'Model'
+    	self.colors = ['blue', 'red', 'green', 'white', 'black']
```

Comment:
So I like that we've limited our options to special values, but what about using an `enum` for these types of things?

https://docs.python.org/3/library/enum.html

We can specify them by value easily (`blueprints.colors.blue` or something) and we don't have to use arrays to store the values. 

This applies to `set_type`, `set_exterior`, `set_interior`, and `set_fuel` as well.